### ISSUE TITLE
Add ability to categorize/group conditional modifiers

### ISSUE THREAD

**Issue by smithkm** [richardwilkes/gcs#282]
Add a category to conditional modifiers that allows similar conditions to be grouped. So for instance all resistances could be grouped, or all conditional attribute bonuses.  Category could be a combo box containing well known categories like Resistances, or Attribute Bonuses as well as any other categories already in use in the file.  

A simple simple version would be to show each category as a group within the conditional modifiers section of the sheet.   A more advanced form would be to allow the user to specify how each category should be handled as a preference on the sheet with options like a separate pane in the sheet, a group in the conditions pane, adding a note to the host trait, or hiding.

**Comment by richardwilkes** [comment]
The latest release added a "conditional modifier" feature, very similar to the reaction modifiers to cover all the other types of modifiers and put them in their own list.

**Comment by smithkm** [comment]
Yep that looks like almost exactly what I was thinking sorry I should have made sure I was up to date before making a ticket.  Being able to assign them to groups instead of just the one big category would still be nice so I'll rewrite the ticket for that.
